modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/src/agent-portal/fragments/fake-content/index.html

```diff
@@ -0,0 +1,5 @@
+<div class="fake-content-container">
+<div class="align-items-center bg-brand-primary-lighten-6 d-flex fake-content-body justify-content-center">
```

Comment:
Ident if missing here

```suggestion
    <div class="align-items-center bg-brand-primary-lighten-6 d-flex fake-content-body justify-content-center">
```